lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -4643,6 +4664,8 @@ def model_apply_prm_baseline_window_to_wall_ratio(model, climate_zone, wwr_build
         std_spc_type = space.additionalProperties.getFeatureAsString('building_type_for_wwr').get
```

Comment:
Will this block of code be used only for stable baseline? If not, will this additional properties fail if it is not a stable baseline?

Reply:
There is a [code block](https://github.com/NREL/openstudio-standards/blob/45193c51165428637294bc36463445486d2f8fb5/lib/openstudio-standards/standards/Standards.Model.rb#L4473-L4484) at the top of this function adds `building_type_for_wwr` to each space. So in here, every space should have this additional property.